hw28_students/js/student.js

```diff
@@ -0,0 +1,228 @@
+//Create Human class
+
+class Human {
+  constructor(args) {
+    this.name = args.name;
+    this.surname = args.surname;
+    this.age = args.age;
+  }
+  getFullName() {
+    return this.name + " " + this.surname;
+  }
+  setFullName(fullName) {
+    return (fullName = [this.name, this.surname]);
+  }
+}
+
+// Create a new instance of the Human class
+
+let human = new Human({
+  name: "Dima",
+  surname: "Dimin",
+  age: 30,
+});
+console.log(human.getFullName());
+console.log(human.setFullName());
+
+// Create a Student class
+
+class Student extends Human {
+  constructor(args) {
+    super(args);
+    this.marks = args.marks;
+  }
+  getAverageMark() {
+    var sum = 0;
+    for (let i = 0; i < this.marks.length; i++) {
+      sum += this.marks[i];
+    }
+    return sum / this.marks.length;
+  }
+  getMinMark() {
+    let min = this.marks[0];
+    for (let i = 0; i < this.marks.length; i++) {
+      if (this.marks[i] < min) {
+        min = this.marks[i];
+      }
+    }
+    return min;
```

Comment:
Please find the other ways to manage it. As an example, you could use `Math.min` method